tests/datadriven/test_variants.py

```diff
@@ -72,14 +72,15 @@ def _compareTwoListFloats(self, a, b):
 
     def _verifyInfoEqual(self, gaObjectInfo, pyvcfInfo):
         def _assertEquivalentGaVCFValues(gaValue, pyvcfValue):
+            compareValue = protocol.getValueFromValue(gaValue)
```

Comment:
I think gaValue if more useful here --- it's telling us it's the GA4GH protocol version of the same value. The ``getValueFromValue`` function name isn't helping much here --- what does it do?